devito/function.py

```diff
@@ -721,6 +743,10 @@ def dt(self):
     @property
     def dt2(self):
         """Symbol for the second derivative wrt the t dimension"""
+        if self.time_order < 2:
+            warning("Time oder smaller than 2, no second time derivatives available," +
+                    " returning 0")
+            return 0
```

Comment:
either use ``else`` here, or remove ``else`` from the other methods, for consistency